gobblin-cluster/src/main/java/org/apache/gobblin/cluster/HelixRetriggeringJobCallable.java

```diff
@@ -266,6 +277,10 @@ private void runJobExecutionLauncher() throws JobException {
 
       try {
         if (planningJobIdFromStore.isPresent() && !canRun(planningJobIdFromStore.get(), planningJobHelixManager)) {
+          TimingEvent timer = new TimingEvent(eventSubmitter, TimingEvent.JOB_SKIP_TIME);
+          HashMap<String, String> metadata = new HashMap<>(Tag.toMap(Tag.tagValuesToString(
+              Lists.newArrayList(HelixUtils.addAdditionalMetadataTags(jobProps, Lists.newArrayList())))));
```

Comment:
sorry if I'm being dense, but why can't you just use the (new) `List` returned, rather than needing to create one more copy?

Reply:
Good catch, I was trying to resolve the compilation issue with this but forget to remove it. Will address